ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -6,6 +6,9 @@ set -o pipefail
 
 echo "************ baremetalds packet setup command ************"
 
+PACKET_PROJECT_ID=$(cat "${CLUSTER_PROFILE_DIR}/packet-project-id")
+PACKET_AUTH_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/packet-auth-token")
```

Comment:
What's the benefit of this approach rather the Ansible variable file? This was avoided as it could increase the chance of leaking, while the `var_files` approach is not subjected to this risk.

Reply:
I think it makes more sense to store the two values as two distinct secrets. Apart from the semantic argument, it's also easier to update one without touching the other.

Reply:
IMHO the drawbacks overwhelm the benefits given from the separated definition (which was something we already did in the past initially) paired with `cat`. It's sufficient to have a `set -x` anywhere previously, added also by mistake, to have an automatic leak (a risk instead avoided thanks to the `var_files`). 

I could see the benefits of a separate definition, thus I'd recommend a safer (and somehow more Ansible oriented) way to load the content of the file in a var, ie:

```
vars:
     packet-project-id: "{{ lookup('file', lookup('env', 'CLUSTER_PROFILE_DIR') + '/packet-project-id') }}"
     packet-auth-token: "{{ lookup('file', lookup('env', 'CLUSTER_PROFILE_DIR') + '/packet-auth-token') }}"
```